### PR TITLE
Added quiet option

### PR DESCRIPTION
**Pull request by aleveillee**
Add quiet support outlined in issue #52.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +121,13 @@ Default: `false`
 
 If the absoluteFilePathsForFormatters option is specified and set to `true`, the file names in the generated reports are absolute.
 
+###### quiet
```
Comment (sindresorhus) [inline README.md]: readme is generated. Needs to be done in `docs/options.md`

File: README.md [hunk 2]
```diff
@@ -121,6 +121,13 @@ Default: `false`
...
+
+Type: `boolean`  
+Default: `false`
+
+Similar to the [CSSLint CLI](https://github.com/CSSLint/csslint/wiki/Command-line-interface#--quiet), if the quiet option is set to `true` then only errors are output to the console.
```
Comment (sindresorhus) [inline README.md]: Too verbose description. Use:

> Only output errors.

File: tasks/csslint.js [hunk 3]
```diff
@@ -82,14 +82,16 @@ module.exports = function(grunt) {
...
+          if( !options.quiet || ( options.quiet && message.type === 'error' )) {
+			  grunt.log.writeln(chalk.red('[') + offenderMessage + chalk.red(']'));
+			  grunt.log[ message.type === 'error' ? 'error' : 'writeln' ](
+				  message.type.toUpperCase() + ': ' +
+				  message.message + ' ' +
+				  message.rule.desc +
+				  ' (' + message.rule.id + ')' +
+				  ' Browsers: ' + message.rule.browsers
+			  );
+          }
```
Comment (sindresorhus) [inline tasks/csslint.js]: Follow the existing codestyle

**Comment by aleveillee** [comment]
Any idea when this new option will be published to npm?

**Pull request closed without merging**